message_ix_models/testing/__init__.py

```diff
@@ -351,3 +353,72 @@ def not_ci(reason=None, action="skip"):
     """
     action = "skipif" if action == "skip" else action
     return getattr(pytest.mark, action)(condition=GHA, reason=reason)
+
+
+def unpack_snapshot_data(context: Context, snapshot_id: int):
+    """Already-unpacked data for a snapshot.
+
+    This copies the .csv.gz files from message_ix_models/data/test/… to the directory
+    where they *would* be unpacked by .model.snapshot._unpack. This causes the code to
+    skip unpacking them, which can be very slow.
+    """
+    if snapshot_id != 0 and snapshot_id != 1:
+        log.info(f"No unpacked data for snapshot {snapshot_id}")
+        return
+
+    dest = context.get_cache_path(
```

Comment:
```python
parts = (f"snapshot-{snapshot_id}", "MESSAGEix-GLOBIOM_1.1-R11_no-policy_baseline")
dest = context.get_cache_path(*parts)
```
Then `parts` can be reused further down instead of repeating.